doc/user/content/sql/system-catalog.md

```diff
@@ -174,6 +281,63 @@ Field                | Type     | Meaning
 `topic`              | [`text`] | The name of the Kafka topic into which the sink is writing.
 `consistency_topic`  | [`text`] | The name of the Kafka topic into which the sink is writing consistency information. This is `NULL` when the sink does not write consistency information.
 
+### `mz_list_types`
+
+The `mz_list_types` table contains a row for each list type in the system.
+
+Field        | Type     | Meaning
+-------------|----------|--------
+`type_id`    | [`text`] | The ID of the list type.
+`element_id` | [`text`] | The IID of the list's element type.
+
+### `mz_message_counts`
+
+The `mz_message_counts` source describes the messages sent and received over the
+dataflow channels in the system.
+
+Field           | Type       | Meaning
+----------------|------------|--------
+`channel`       | [`bigint`] | The ID of the channel. Corresponds to [`mz_dataflow_channels.id`](#mz_dataflow_channels).
+`source_worker` | [`bigint`] | The ID of the worker thread sending the message.
+`target_worker` | [`bigint`] | The ID of the worker thread receiving the message.
+`sent`          | [`bigint`] | The number of messages sent.
+`received`      | [`bigint`] | The number of messages received.
+
+### `mz_materialization_dependencies`
+
+The `mz_materialization_dependencies` source describes the sources that each
+[dataflow] depends on.
+
+Field      | Type       | Meaning
+-----------|------------|--------
+`dataflow` | [`text`]   | The ID of the index that created the dataflow. Corresponds to [`mz_indexes.id`](#mz_indexes).
+`source`   | [`text`]   | The ID of the source. Corresponds to [`mz_sources.id`](#mz_sources).
+`worker`   | [`bigint`] | The ID of the worker thread hosting the dataflow.
+
+### `mz_materialization_frontiers`
+
+The `mz_materialization_frontiers` source describes the frontier for each
```

Comment:
nit, but I think this is a view, not a source. It derives from `mz_worker_materialization_frontiers` which is the source.

Reply:
Ooh thanks!